webpack.common.js

```diff
@@ -155,6 +155,9 @@ module.exports = {
         REACT_APP_RECAPTCHA_SITE_KEY: JSON.stringify(
           process.env.REACT_APP_RECAPTCHA_SITE_KEY || "",
         ),
+        REACT_APP_SINGLE_TENANT_MODE: JSON.stringify(
+          process.env.REACT_APP_SINGLE_TENANT_MODE || "",
+        ),
```

Comment:
I'm confused. Wasn't this supposed to be a boolean?

Reply:
values in `process.env` are always strings even if the env var has `true` or `false` set. Any value that's not `true` set to `process.env.REACT_APP_SINGLE_TENANT_MODE` will default to `false` when we generate the env configs in app!